test/Sync.t.sol

```diff
@@ -168,7 +198,7 @@ contract SyncTest is Test, Deployers, GasSnapshot {
         params[6] = abi.encode(currency0, address(this), manager, 10);
 
         actions[7] = Actions.SETTLE;
-        params[7] = abi.encode(currency0);
+        params[7] = abi.encode(address(router));
```

Comment:
mmmmm feels kind of annoying that you have to figure out who the msg.sender's address is going to be and pass it in? what if theres a scenario where yu dont know who's going to be executing your trade?
i feel like it would be nice to have a default of msg.sender somehow. Like if someone passes in address(0), default of msg.sender? Or have a separate function with no parameters.

Reply:
added `settleFrom(address)`